app_check/src/include/firebase/app_check/debug_provider.h

```diff
@@ -43,6 +49,10 @@ class DebugAppCheckProviderFactory : public AppCheckProviderFactory {
 
  private:
   DebugAppCheckProviderFactory();
+
+  /// @cond FIREBASE_APP_INTERNAL
```

Comment:
Similar to the above, if unconditional forward declaration is possible, then this variable can be defined unconditionally as well.

Reply:
Same as above, will remove.